go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/ipfs/go-datastore v0.1.1
 	github.com/ipfs/go-log v0.0.1
 	github.com/keep-network/go-libp2p-bootstrap v0.0.0-20190611114437-e92bd71e8199
-	github.com/keep-network/keep-common v0.1.1-0.20200330215727-8ca95ff9de56
+	github.com/keep-network/keep-common v0.1.1-0.20200407084635-41258118ebd7
```

Comment:
NOTE! this is not a master branch dependency. Need to merge [keep common PR](https://github.com/keep-network/keep-common/pull/24) first and then update this version.

Reply:
Done in https://github.com/keep-network/keep-core/pull/1550/commits/447538e8a9de2affae013251f513e7d9a552b11f